overlays/default.nix

```diff
@@ -17,8 +17,23 @@
     };
   })
 
-  # Patched and override existing name because of it is not cofigurable
+  # Patched and override existing name because of it is not configurable
   (final: prev: {
+    # https://github.com/NixOS/nixpkgs/blob/nixos-25.05/pkgs/by-name/li/linux-firmware/package.nix
+    linux-firmware = prev.linux-firmware.overrideAttrs (
```

Comment:
Adding to the hardware configuration may be enough. Using an overlay seems too much. 🤔